deploy/hooks/post-restore

```diff
@@ -8,4 +8,19 @@ if [ -x /etc/init.d/apache2 ]; then
     sudo /etc/init.d/apache2 restart
 fi
 
-exit 0
```

Comment:
yes, I could bail out with error directly when apache status is wrong. Restart is blocking.

And only wait 70 when apache was/is up.